src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -43,15 +44,15 @@ def execute(args):
 
     testcases.append(testcase)
```

Comment:
`testcase` should only be appended if modified above - no need to `put` it otherwise.

Reply:
Done. Thanks for the catch!